### DISCUSSION TITLE
docs/home/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#118] [Announcements]
# docs/home/

The scope of this project is to create a kubernetes cluster at home using Raspberry Pis and to automate its deployment and configuration applying IaC (infrastructure as a code) and GitOps methodologies with tools like Ansible and ArgoCD. How to automatically deploy K3s baesed kubernetes cluster, Longhorn as distributed block storage for PODs’ persistent volumes, Prometheus as monitoring solution, EFK+Loki stack as centralized log management solution, Velero and Restic as backup solution and Linkerd as service mesh architecture.

https://picluster.ricsanfre.com/docs/home/

**Comment by bpartin2009** [comment]
Job well done....... Your project is well documented and I'm starting a copy of your hybrid setup and using your documentation as a guide. We use vCenter/vSphere, Horizon VDI, and Ansible Automation Platform/Hub at work but looking to implement Hashicorp Vault and Containerizing service that we currently run on VMs. I have two employee getting Kubernetes certified but I need to educate myself on my home lab as a lead. 
What you've done is no small task. Good ob.

**Reply by ricsanfre** [reply]
Thanks

**Comment by irazashaikh992** [comment]
Backups are like insurance policies for your data - you hope you never need them, but when you do, you'll be glad you have them!"

**Comment by marmila** [comment]
Amazing Ricardo, congrats. I never get tired of thanking you for your efforts.

**Reply by ricsanfre** [reply]
Thanks Marco
